vm/isolated/isolated.go

```diff
@@ -173,26 +182,38 @@ func (inst *instance) ssh(command string) error {
 
 func (inst *instance) repair() error {
 	log.Logf(2, "isolated: trying to ssh")
-	if err := inst.waitForSSH(30 * time.Minute); err == nil {
+	if err := inst.waitForSSH(3 * time.Minute); err == nil {
+		log.Logf(2, "isolated: ssh succeeded")
+	} else {
+		log.Logf(2, "isolated: ssh failed")
 		if inst.cfg.TargetReboot {
 			log.Logf(2, "isolated: trying to reboot")
-			inst.ssh("reboot") // reboot will return an error, ignore it
-			if err := inst.waitForReboot(5 * 60); err != nil {
+			if inst.cfg.USBDevNum == "" {
+				inst.ssh("reboot")
+			} else {
+				_, err = exec.Command("/bin/sh", "-c", "echo 0 > /sys/bus/usb/devices/"+inst.cfg.USBDevNum+"/authorized").Output()
```

Comment:
We don't need to start 2 subprocesses, full fledged shell and rely on particular layout of helper binaries in the image to invoke write system call. That's over-complicated and overly-fragile way to achieve that. Is there even /bin/sh on windows? ;)
Use osutil.WriteFile (or os.File directly) to write to a file.

Reply:
Great idea. Actually, I'm not familiar with Go lang.
Thank you for good idea.